gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -268,6 +427,7 @@ def main(args):
         tmp_dir="gs://gnomad-tmp-4day",
     )
     v4_genome_release_resources = get_v4_genomes_release_resources(overwrite=overwrite)
+    v3_meta_ht = v4_genome_release_resources.meta_ht.ht()
```

Comment:
```suggestion
    meta_ht = v4_genome_release_resources.meta_ht.ht()
```
I found v3_meta_ht here is confusing, I know here you're using as a global, but it is in other two args too. I prefer 'meta_ht' to be reserved for HGDP/TGP meta only.